src/EditorFeatures/VisualBasicTest/Structure/TryBlockStructureTests.vb

```diff
@@ -12,36 +12,130 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Outlining
             Return New TryBlockStructureProvider()
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Outlining)>
+        <Fact, Trait(Traits.Feature, Traits.Features.Outlining), Trait(Traits.Feature, Traits.Features.AdditionalInternalStructureOutlinings)>
```

Comment:
what's this extra trait for?

Reply:
So the test explorer produces an additional section, just containing the updated providers.